tests/controller/deletecontent.js

```diff
@@ -233,12 +233,6 @@ describe( 'DataController', () => {
 				'<heading1>f[]</heading1><paragraph>x</paragraph>'
 			);
 
-			test(
```

Comment:
It's better to avoid deleting tests completely. This may be the only test which checks if the middle paragraph (foo) is deleted. So, either just adjust its result or add some content outside to not make it a full content selection (and adjust the title).